### ISSUE TITLE
Get the request data via the Alma API

### ISSUE THREAD

**Issue by SydneyUni-Jim** [SydneyUniLibrary/print-slip-report#11]
- [x] Make a request to the [get requested resources](https://developers.exlibrisgroup.com/alma/apis/docs/tasklists/R0VUIC9hbG1hd3MvdjEvdGFzay1saXN0cy9yZXF1ZXN0ZWQtcmVzb3VyY2Vz/) API with the location code and circ desk code from #10.
- [x] Show a spinner while the API request is in progress
- [x] Show an alert for general errors
- [x] On error code 40166410 where the message starts with "The parameter library is invalid.", show a mat-error element on the library text field with the valid options from ```message``` of the API result.
- [x] On error code 40166410 where the message starts with "The parameter circ_desk is invalid.", show a mat-error element on the circ_desk text field with the valid options from ```message``` of the API result.

We can assume the location code will be valid.

**Comment by SydneyUni-Jim** [comment]
Example of error code 40166410

```json
{
  "ok": false,
  "status": 400,
  "statusText": "Bad Request",
  "message": "The parameter circ_desk is invalid. Received: DEFAULT_CIRC_DESK2. Valid options are: [DEFAULT_CIRC_DESK,FET_CLOUD,FSH_ALL,FSH_HAL_03,FSH_LIBAPP,FSH_MAIL,FSH_RETURN,FSH_SA_01,FSH_SC2H_3,FSH_SC2H_4,FSH_SC_01,FSH_SC_02,FSH_SORTER].",
  "error": {
    "errorsExist": true,
    "errorList": {
      "error": [
        {
          "errorCode": "40166410",
          "errorMessage": "The parameter circ_desk is invalid. Received: DEFAULT_CIRC_DESK2. Valid options are: [DEFAULT_CIRC_DESK,FET_CLOUD,FSH_ALL,FSH_HAL_03,FSH_LIBAPP,FSH_MAIL,FSH_RETURN,FSH_SA_01,FSH_SC2H_3,FSH_SC2H_4,FSH_SC_01,FSH_SC_02,FSH_SORTER].",
          "trackingId": "E01-1903074731-ZYIIU-AWAE490532345"
        }
      ]
    },
    "result": null
  }
}
```

**Comment by SydneyUni-Jim** [comment]
We only have a maximum of 100 requested items. This is the most we can get in a single API call. This will be addressed in #27.
